api/README.md

```diff
@@ -0,0 +1,20 @@
+# Flask Back-end
+An application that serves as a back-end to our app.
+
+## Setting Up
+Before setting up, you should have Python 3 and pipenv installed globally.
+
```

Comment:
Consider adding the following to the description:
From the command line, go to the directory that contains the Pipfiles (Currently "api", if you make the change then, "semester-project-team-2").
```bash
cd path-to-directory
```

Reply:
Will add to wiki